### ISSUE TITLE
Update issue template.

### ISSUE THREAD

**Issue by lolgear** [CocoaLumberjack/CocoaLumberjack#1168]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

It seems that current template is outdated. Also, GitHub suggests various issue templates for bugs and features.
I think it is time to update issue templates.
